frontend/src/pages/AccountSend.vue

```diff
@@ -600,15 +600,20 @@ function useSendForm() {
 
   // Batch Send Computed Form Parameters
   const batchSendHumanTotalAmount = computed(() => {
-    const ethTotal = summaryAmount.value.get(NATIVE_TOKEN.value);
+    let ethTotal;
+    for (const token of summaryAmount.value.keys()) {
+      if (token.symbol === NATIVE_TOKEN.value.symbol) {
+        ethTotal = summaryAmount.value.get(token);
+      }
+    }
```

Comment:
why was this changed? isn't calling `.get` for a only single condition equivalent to what we had if we tweaked it like this:

```suggestion
const ethTotal = summaryAmount.value.get(NATIVE_TOKEN.value) || '0'
```

Reply:
This is actually relevant to this https://github.com/ScopeLift/umbra-protocol/pull/528#discussion_r1218364190 to `.get` you need to provide a `TokenInfoExtended` and `NATIVE_TOKEN` isn't one. 

And casting it as `unknown` as `TokenInfoExtended` didn't work because it looks like the `LogoURI` is different between `NATIVE_TOKEN` and the selected native token. I tried changing that to be the same but still it was `NATIVE_TOKEN` couldn't be used to get from the `map.` 

Felt like it wasn't worth it to further investigate and this was my solution because we know the native token's `symbol` will be the same.